lang/lang_en_cz.txt

```diff
@@ -1240,4 +1240,196 @@
 
 #MSG_OFF
 " [off]"
+" [vyp]"
+
+#
+"Back"
+"Zpet"
+
+#
+"Checks"
+"Kontrola"
+
+#
+"False triggering"
+"Falesne spusteni"
+
+#
+"FINDA:"
+"\x00"
+
+#
+"Firmware   [none]"
+"Firmware  [Zadne]"
+
+#
+"Firmware [strict]"
+"Firmware [Prisne]"
+
+#
+"Firmware   [warn]"
+"Firmware[Varovat]"
+
+#
+"HW Setup"
+"HW nastaveni"
+
+#
+"IR:"
 "\x00"
+
+#
+"Magnets comp.[N/A]"
+"Komp. magnetu[N/A]"
+
+#
+"Magnets comp.[Off]"
+"Komp. magnetu[Vyp]"
+
+#
+"Magnets comp. [On]"
+"Komp. magnetu[Zap]"
+
+#
+"Mesh         [3x3]"
+"Mesh         [3x3]"
+
+#
+"Mesh         [7x7]"
+"Mesh         [7x7]"
+
+#
+"Mesh bed leveling"
+"Mesh Bed Leveling"
+
+#
+"MK3S firmware detected on MK3 printer"
+"MK3S firmware detekovan na tiskarne MK3"
+
+#
+"MMU Mode [Normal]"
+"MMU mod  [Normal]"
+
+#
+"MMU Mode[Stealth]"
+"MMU Mod   [Tichy]"
+
+#
+"Mode change in progress ..."
+"Probiha zmena modu..."
+
+#
+"Model      [none]"
+"Model     [Zadne]"
+
+#
+"Model    [strict]"
+"Model    [Prisne]"
+
+#
+"Model      [warn]"
+"Model   [Varovat]"
+
+#
+"Nozzle d.  [0.25]"
+"Tryska     [0.25]"
+
+#
+"Nozzle d.  [0.40]"
+"Tryska     [0.40]"
+
+#
+"Nozzle d.  [0.60]"
+"Tryska     [0.60]"
+
+#
+"Nozzle     [none]"
+"Tryska    [Zadne]"
+
+#
+"Nozzle   [strict]"
+"Tryska   [Prisne]"
+
+#
+"Nozzle     [warn]"
+"Tryska  [Varovat]"
+
+#
+"G-code sliced for a different level. Continue?"
+"\x00"
+
+#
+"G-code sliced for a different level. Please re-slice the model again. Print cancelled."
+"\x00"
+
+#
+"G-code sliced for a different printer type. Continue?"
+"G-code je pripraven pro jiny typ tiskarny. Pokracovat?"
+
+#
+"G-code sliced for a different printer type. Please re-slice the model again. Print cancelled."
+"G-code je pripraven pro jiny typ tiskarny. Prosim preslicujte model znovu. Tisk zrusen."
+
+#
+"G-code sliced for a newer firmware. Continue?"
+"G-code je pripraven pro novejsi firmware. Pokracovat?"
+
+#
+"G-code sliced for a newer firmware. Please update the firmware. Print cancelled."
+"G-code je pripraven pro novejsi firmware. Prosim aktualizujte firmware. Tisk zrusen."
+
+#
+"PINDA:"
+"\x00"
+
+#
+"Preheating to cut"
+"Predehrev k ustrizeni"
+
+#
+"Preheating to eject"
+"Predehrev k vysunuti"
+
+#
+"Printer nozzle diameter differs from the G-code. Continue?"
+"Prumer trysky tiskarny se lisi od G-code. Pokracovat?"
+
+#
+"Printer nozzle diameter differs from the G-code. Please check the value in settings. Print cancelled."
+"Prumer trysky tiskarny se lisi od G-code. Prosim zkontrolujte nastaveni. Tisk zrusen."
+
+#
+"Rename"
+"Prejmenovat"
+
+#
+"Select"
+"Vybrat"
+
+#
+"Sensor info"
+"Senzor info"
+
+#
+"Sheet"
+"Plat"
+
+#
+"Sound     [assist]"
```

Comment:
After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the `not_tran.txt` showed that `Sound    [assist]` is missing.
Checking the `lang_en.txt` and `lang_en_??.txt` it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between `Sound` and `[assist]`